src/App.tsx

```diff
@@ -1,8 +1,74 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
+import { TodoList } from './TodoList/TodoList';
+import { getTodo, getUser } from './api/api';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC = () => {
+  const [todos, setTodo] = useState<TodoWithUser[]>([]);
+  const [isLoaded, setLoading] = useState(false);
+
+  const loadTodo = () => {
+    setLoading(true);
+    Promise.all([getTodo(), getUser()])
+      .then(response => {
```

Comment:
Деструктурируй прямо тут response
```suggestion
      .then(([])=> {
```